routes/items.py

```diff
@@ -67,6 +68,33 @@ def remove_item(code):
     return redirect(url_for('admin.stocks'))
 
 
+@items.route('/items/edit/<code>', methods=['POST'])
+def Edit(code):
+    if session['privilege'] == 3:
```

Comment:
Within the user stories, Staff members are allowed to edit stock as opposed to just the admins. Does the user story need to change or privileges?